src/main/java/com/github/onsdigital/thetrain/configuration/AppConfiguration.java

```diff
@@ -22,34 +23,44 @@
     public static final String THREAD_POOL_SIZE_ENV_KEY = "PUBLISHING_THREAD_POOL_SIZE";
     public static final String PORT_ENV_KEY = "PORT";
     public static final String ENABLE_VERIFY_PUBLISH_CONTENT = "ENABLE_VERIFY_PUBLISH_CONTENT";
-
-    private static final String THREAD_POOL_SIZE_PARSE_ERR = format("configuration value %s could not be parsed to " +
-            "Integer", THREAD_POOL_SIZE_ENV_KEY);
-
-    private static final String PORT_PARSE_ERR = format("configuration value %s could not be parsed to Integer",
-            PORT_ENV_KEY);
+    public static final String FILE_UPLOADS_TMP_DIR = "FILE_UPLOADS_TMP_DIR";
+    public static final String MAX_FILE_UPLOAD_SIZE_MB = "MAX_FILE_UPLOAD_SIZE_MB";
+    public static final String MAX_REQUEST_SIZE_MB = "MAX_REQUEST_SIZE_MB";
+    public static final String FILE_THRESHOLD_SIZE_MB = "FILE_THRESHOLD_SIZE_MB";
 
     private Path transactionStore;
     private Path websitePath;
+    private Path fileUploadsTmpDir;
     private int publishThreadPoolSize;
     private int port;
     private boolean enableVerifyPublish;
+    private long maxFileUploadSize;
+    private long maxRequestSize;
+    private int fileThresholdSize;
 
     /**
      * @throws ConfigurationException
      */
     private AppConfiguration() throws ConfigurationException {
         this.transactionStore = loadTransactionStoreConfig();
         this.websitePath = loadWebsitePathConfig();
-        this.publishThreadPoolSize = loadPublishPoolSizeConfig();
-        this.port = loadPortConfig();
         this.enableVerifyPublish = loadEnableVerifyPublishContentFeatureFlag();
+        this.fileUploadsTmpDir = createTmpFileUploadsDir();
+        this.publishThreadPoolSize = getIntegerEnvVar(THREAD_POOL_SIZE_ENV_KEY);
+        this.port = getIntegerEnvVar(PORT_ENV_KEY);
+        this.maxFileUploadSize = getLongEnvVar(MAX_FILE_UPLOAD_SIZE_MB);
+        this.maxRequestSize = getLongEnvVar(MAX_REQUEST_SIZE_MB);
+        this.fileThresholdSize = getIntegerEnvVar(FILE_THRESHOLD_SIZE_MB);
 
         info().data(TRANSACTION_STORE_ENV_KEY, transactionStore)
                 .data(WEBSITE_ENV_KEY, websitePath)
                 .data(THREAD_POOL_SIZE_ENV_KEY, publishThreadPoolSize)
                 .data(PORT_ENV_KEY, port)
                 .data(ENABLE_VERIFY_PUBLISH_CONTENT, enableVerifyPublish)
+                .data(FILE_UPLOADS_TMP_DIR, fileUploadsTmpDir)
+                .data(MAX_FILE_UPLOAD_SIZE_MB, maxFileUploadSize)
+                .data(MAX_REQUEST_SIZE_MB, maxRequestSize)
+                .data(FILE_THRESHOLD_SIZE_MB, FileUtils.byteCountToDisplaySize(fileThresholdSize))
```

Comment:
I think we want to apply `FileUtils.byteCountToDisplaySize` to `MAX_FILE_UPLOAD_SIZE_MB` and `MAX_REQUEST_SIZE_MB`, as they are byte values? If so, I don't think it needs to be applied to `FILE_THRESHOLD_SIZE_MB`, as it is a MB value